Mage.Sets/src/mage/cards/l/LethalThrowdown.java

```diff
@@ -24,7 +24,7 @@
  */
 public final class LethalThrowdown extends CardImpl {
 
-    private static final FilterPermanent filter = new FilterPermanent("sacrifice a modified creature");
+    private static final FilterPermanent filter = new FilterPermanent("a modified creature");
```

Comment:
Must be CreaturePermanent, not Permanent

Reply:
ad66b7439bb1c1c2df4767e0d6b9db84d1d9b522 fixed in that commit.